routes/friend.js

```diff
@@ -0,0 +1,62 @@
+var express = require('express');
+var router = express.Router();
+const friend = require('../models/friends');
+
+/* GET home page. */
+router.get("/requests", function(req, res, next) {
+    friend.find({friend1:req.user._id, confirmed: false}).exec(function (err, own_requests) {
+        if(err) return res.status(500).json({message:"an error occured "+err});
+       friend.find({friend2:req.user._id, confirmed: false}).exec(function (err, requests) {
+           if(err) return res.status(500).json({message:"an error occured "+err});
+           res.json({own_requests: own_requests, requests: requests});
+       })
+    });
+});
+
+router.get("/allconnections", function(req, res, next) {
+    friend.find({$or:[{friend1: req.user._id},{friend2:req.user._id}]}).exec(function (err, connections) {
+        if(err) return res.status(500).json({message:"an error occured "+err});
+        res.json({connections:connections});
+    });
+});
+
+router.post("/dorequest", function(req,res,next){
```

Comment:
werkwoorden horen niet thuis in een REST api

Reply:
k